src/amvp_util.c

```diff
@@ -1024,7 +1024,7 @@ JSON_Object *amvp_get_obj_from_rsp(AMVP_CTX *ctx, JSON_Value *arry_val) {
     }
     reg_array = json_value_get_array(arry_val);
 
```

Comment:
I think originally we pulled the version out as well and checked it, but I guess we don't care what's there since this will get changed back once the server removes it, correct ?

Reply:
Thats my thought, yea. If we end up going a different route we can add the check back in.